Doc/howto/logging.rst

```diff
@@ -1078,20 +1078,21 @@ need more precise control over what logging information is collected. Here's a
 list of things you can do to avoid processing during logging which you don't
 need:
 
-+-----------------------------------------------+----------------------------------------+
-| What you don't want to collect                | How to avoid collecting it             |
-+===============================================+========================================+
-| Information about where calls were made from. | Set ``logging._srcfile`` to ``None``.  |
-|                                               | This avoids calling                    |
-|                                               | :func:`sys._getframe`, which may help  |
-|                                               | to speed up your code in environments  |
-|                                               | like PyPy (which can't speed up code   |
-|                                               | that uses :func:`sys._getframe`).      |
-+-----------------------------------------------+----------------------------------------+
-| Threading information.                        | Set ``logging.logThreads`` to ``0``.   |
-+-----------------------------------------------+----------------------------------------+
-| Process information.                          | Set ``logging.logProcesses`` to ``0``. |
-+-----------------------------------------------+----------------------------------------+
++-----------------------------------------------+---------------------------------------------------+
+| What you don't want to collect                | How to avoid collecting it                        |
++===============================================+===================================================+
+| Information about where calls were made from. | Set ``logging._srcfile`` to ``None``.             |
+|                                               | This avoids calling :func:`sys._getframe`, which  |
+|                                               | may help to speed up your code in environments    |
+|                                               | like PyPy (which can't speed up code that uses    |
+|                                               | :func:`sys._getframe`).                           |
++-----------------------------------------------+---------------------------------------------------+
+| Threading information.                        | Set ``logging.logThreads`` to ``False``.          |
++-----------------------------------------------+---------------------------------------------------+
+| Process information.                          | Set ``logging.logProcesses`` to ``False``.        |
++-----------------------------------------------+---------------------------------------------------+
+| Multiprocess information.                     | Set ``logging.logMultiprocessing`` to ``False``.  |
++-----------------------------------------------+---------------------------------------------------+
```

Comment:
I think it might not be clear from this what the difference is between "process information" and "multiprocess information".   We could say more precisely "process id" instead of "process information" and "process name" instead of "multiprocessing information".

But then as a reader I might wonder why they are handled separately. So how about 

| Process Id  | Set ``logging.logThreads`` to ``False``. |
+------------------------------------------------------ ...
| Process Name (requires the multiprocessing module) |  Set ``logging.logMultiprocessing`` to ``False``.  |

Reply:
This documentation difficulty is actually a reflection of the API being a bit clumsy. I guess it evolved into its current state, but if we were to do it from scratch we would probably make it something like:

logProcesses = False suppresses both processId and processName logging.
rename logMultiprocessing to logProcessName?

Is it worth reviewing this API before documenting it?

Reply:
I don't think we should make any functional changes here, just documentation changes. Changing "Process information" to "Current process information" or "Current process ID" would help distinguish from the multiprocessing case.